examples/api/lib/material/slider/slider.1.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
You'll need to add a test for this example. Take a look at the other example tests for how to do them (they're pretty simple).

Reply:
I might have mistaken but can you point me wherer this example is used in docs?

Reply:
added in latest commit